api.md

```diff
@@ -0,0 +1,185 @@
+# Submit a Claim
+
+## Request
+
+`POST <Base URL>/entries`
+
+## Response
+
+One of the following
+- Status 202 - Registration is pending
+    - Header: `Location: <Base URL>/operations/<Operation ID>`
+    - Header: `Content-Type: application/json`
+    - Body: `{ "operationId": "<Operation ID>", "status": "pending" }`
+
+- Status 400 - Registration was unsuccessful due to invalid input
+    - Error code `InvalidInput`
+
+- Status 201 - Registration was successful
```

Comment:
I think it's better to have 202 always and if it's completed immediately, then the status is `"registered"`.